rpc/flipt/audit/event.proto

```diff
@@ -0,0 +1,34 @@
+syntax = "proto3";
+
+package flipt.audit;
+
+option go_package = "go.flipt.io/flipt/rpc/flipt/audit";
+
+import "google/protobuf/timestamp.proto";
+
+message Actor {
+  string authentication = 1;
+  string ip = 2;
+  string email = 3;
+  string name = 4;
+  string picture = 5;
+}
+
+message Metadata {
+    optional Actor actor = 1;
+}
+
+message Event {
+    string version = 1;
+    string type = 2;
+    string action = 3;
+    Metadata metadata = 4;
+    optional string payload = 5;
+    google.protobuf.Timestamp timestamp = 6;
+    optional string status = 7;
+}
+
```

Comment:
not sure if we need this service tbh, probably just need the protos/messages

If we do end up needing this service, you can add the following comment above it like:

```
// flipt:sdk:ignore
service AuditEventService {
```

This will prevent our codegen from generating the Go SDK code for it, which I don't think we need

But yeah I don't think we need the actual GRPC service at all tbh

Reply:
That service is a workaround with GRPC generation. Somehow I isn't able to generate sources if there is no service or //ignore comment present. It fails with panic

```sh
Generating proto files...
panic: runtime error: index out of range [0] with length 0
```

The magic is going here
https://github.com/flipt-io/flipt/blob/7d36166b21a199ec788fff9e0bc70093c939bae5/internal/cmd/protoc-gen-go-flipt-sdk/main.go#L84-L86
If `file.Services` contains no elements the code still want to get the first element.

 I've addressed it in #3215